tools/go-cli/go-whisk-cli/commands/rule.go

```diff
@@ -180,11 +172,9 @@ var ruleCreateCmd = &cobra.Command{
         var err error
         var shared bool
 
-        if len(args) != 3 {
-            whisk.Debug(whisk.DbgError, "Invalid number of arguments: %d\n", len(args))
-            errStr := fmt.Sprintf("Invalid number of arguments (%d) provided; exactly three arguments are expected", len(args))
-            werr := whisk.MakeWskError(errors.New(errStr), whisk.EXITCODE_ERR_GENERAL, whisk.DISPLAY_MSG, whisk.DISPLAY_USAGE)
-            return werr
+        if whiskErr := checkArgs(args, 3, 3, "Rule create",
+            "A rule, trigger and action name is required."); whiskErr != nil {
```

Comment:
Change to "A rule name, trigger name and action name are [all] required."   Not sure is having the `all` makes it clearer or not.